pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 "Tracker" = "https://github.com/N-Wouda/ALNS/issues"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8, <4.0"
 numpy = ">=1.15.2"
 matplotlib = ">=2.2.0"

```

Comment:
Should we also bump the versions of `numpy` and `matplotlib`?

Reply:
That's `numpy >= 1.18.0`, since `matplotlib >= 2.2.0` already seems to be OK with Py3.8. I've bumped numpy.